numpy/lib/twodim_base.py

```diff
@@ -373,6 +386,11 @@ def tri(N, M=None, k=0, dtype=float):
     return m
 
 
+def _tril_dispatcher(m, k=None):
```

Comment:
Maybe call it `triul_dispatcher`, since it is used for both?

Reply:
done